tests/CMakeLists.txt

```diff
@@ -353,6 +353,7 @@ add_test_case(h2_client_stream_send_data_controlled_by_stream_window_size)
 add_test_case(h2_client_stream_send_data_controlled_by_negative_stream_window_size)
 add_test_case(h2_client_stream_send_data_controlled_by_connection_window_size)
 add_test_case(h2_client_stream_send_data_controlled_by_connection_and_stream_window_size)
+add_test_case(h2_client_stream_send_window_update)
```

Comment:
add these todos please

#TODO add_test_case(h2_client_stream_err_flow_control)
#TODO add_test_case(h2_client_conn_err_flow_control)

we can't actually test this right now, since we always auto-update and can't even tweak INITIAL_WINDOW_SIZE and MAX_FRAME_SIZE yet to make it possible 🙃